src/main/java/ch/uzh/ifi/hase/soprafs22/controller/GameController.java

```diff
@@ -51,9 +50,21 @@ public List<GameGetDTO> getGames() {
      * @param id
      * @return GameGetDTO
      */
-    @GetMapping(path = "/games/{id}")
-    public GameGetDTO getGameById(@PathVariable String id) {
-        Game game = gameService.getGameById(id);
+    @GetMapping(path = "/games/{gameToken}")
+    public GameGetDTO getGameByToken(@PathVariable("gameToken") String gameToken) {
+        Game game = gameService.getGameByToken(gameToken);
+        return DTOMapper.INSTANCE.convertEntityToGameGetDTO(game);
+    }
+
+    /**
+     * Add player to a game by id
+     * @param gameToken
+     * @param userToken
+     * @return GameGetDTO
+     */   
+    @PutMapping(path = "/games/{gameToken}/player/{userToken}")
+    public GameGetDTO addPlayerToGame(@PathVariable("gameToken") String gameToken, @PathVariable("userToken") String userToken) {
+        Game game = gameService.addPlayerToGame(gameToken, userToken);
         return DTOMapper.INSTANCE.convertEntityToGameGetDTO(game);
```

Comment:
I think a put request doesnt need a return value. So i would do a void function and a 204 status code...